tools/report.collector.in

```diff
@@ -284,6 +288,8 @@ sanitize() {
     touch -r $file $ref  # save the mtime
     if [ "`basename $file`" = ha.cf ]; then
 	sanitize_hacf
+    elif [ "$file" = "$DRBD_INFO_F" ]; then
+	$decompress | sanitize_drbd | $compress
```

Comment:
This certainly makes sense with the current code structure, but I'm thinking it would be more secure to pipe drbdadm dump through sanitize_drbd when saving the file originally. That would ensure the secret is never saved to disk, and there's no risk of exposing it if the compression command fails for whatever reason.